react/features/toolbox/components/StatelessToolbar.web.js

```diff
@@ -0,0 +1,65 @@
+/* @flow */
+
+import React, { Component } from 'react';
+
+/**
+ * Implements a toolbar in React/Web. It is a strip that contains a set of
+ * toolbar items such as buttons. Toolbar is commonly placed inside of a
+ * Toolbox.
+ *
+ * @class Toolbar
```

Comment:
I think these comments need updating, at least redoing the "class Toolbar" part.